src/fontra/views/editor/editor.js

```diff
@@ -1411,9 +1411,41 @@ export class EditorController {
       { title: "Cancel", isCancelButton: true },
       { title: "Add", isDefaultButton: true, result: "ok", disabled: true },
     ]);
-    dialog.setContent(glyphsSearch);
+    dialog.setContent(
+      html.div(
+        {
+          style: `
+          grid-row: 2 / -1;
+          display: flex;
+          flex-direction: column;
+        `,
+        },
+        [
+          glyphsSearch,
+          html.div({ class: "dialog-add-to-all-sources-checkbox" }, [
```

Comment:
Suggest to not specify a class if we don't need it:
```suggestion
          html.div({}, [
```